util/generate-ami-list.py

```diff
@@ -52,7 +53,7 @@ def get_ami_list(regions, date, version):
 
         except SystemExit:
             sys.exit("Error: there are no AMIs in the selected region (%s)" % region_name)
```

Comment:
@sean-smith I tested the code and it works. 
We need to make some changes, because _boto3_ now supports _ap-northeast-3_ and _eu-west-3_ where we don't have yet AMIs.

To make it work, you should remove the following lines:
```
 except SystemExit:
             sys.exit("Error: there are no AMIs in the selected region (%s)" % region_name)
```
and just change the rows before with:
```
 if len(amis) == 0:
                 print("Warning: there are no AMIs in the selected region (%s)" % region_name)
```
in this way the cycle goes ahead skipping the new two regions.
Then you can remove the `TODO test` and the old `boto2 `inclusion on the top.

Reply:
fixed